uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/device/settings/GoogleSettings.java

```diff
@@ -168,8 +176,8 @@ public void forwardDeviceTimeForOneDay() {
 
             final Calendar cal = Calendar.getInstance();
 
-            // add one to move to next day
-            cal.add(Calendar.DATE, 1);
+            // add the # of seconds to forward device time
+            cal.add(Calendar.SECOND, seconds);
```

Comment:
So this code change works with all the existing UI Automation?

Reply:
It should, is there a reason why you see it's not?